Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -516,6 +773,19 @@ extension NewTabPageViewController: UICollectionViewDelegateFlowLayout {
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
         sections[section].collectionView?(collectionView, layout: collectionViewLayout, minimumInteritemSpacingForSectionAt: section) ?? 0
     }
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+        sections[section].collectionView?(collectionView, layout: collectionViewLayout, referenceSizeForHeaderInSection: section) ?? .zero
+    }
+}
+
+// MARK: - UISearchControllerDelegate
+extension NewTabPageViewController: UISearchControllerDelegate {
```

Comment:
What is this `UISearchControllerDelegate` for in NTP?

Reply:
I… do not know, must have been a auto-complete typo. Was supposed to be `UICollectionViewDelegate`